src/main/java/duke/Event.java

```diff
@@ -0,0 +1,27 @@
+package duke;
+
+/**
+ * duke.Event class to create Events for the duke list
```

Comment:
I think that you are missing full stops for the JavaDocs; similarly for the rest of the code.